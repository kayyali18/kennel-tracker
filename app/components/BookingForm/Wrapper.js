import styled from 'styled-components'

const Form1 = styled.section`
  color: #fff;

  max-height: 70vh;

  overflow: auto;

  input {
    border: 2px solid black;
  }

  .form-enter {
    margin-top: 30px;
    opacity: 0.01;
  }
  .form-enter-active {
    margin-top: 0px;
    opacity: 1;
    transition: opacity 0.5s ease, margin 0.5s ease;
  }

  .form-leave {
    margin-top: 0px;
    opacity: 1;
  }
  .form-leave-active {
    margin-top: -30px;
    opacity: 0.01;
    transition: opacity 0.3s ease, margin 0.5s ease;
  }
`
export default Form1

// make them checkboxes so when checked they can move to the next
