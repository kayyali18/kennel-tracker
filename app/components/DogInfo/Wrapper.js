import styled from 'styled-components'

const Wrapper = styled.section`
  height: 100vh;
  max-width: 100vw;
  background: linear-gradient(#a5a5a5, #f2f2f2);
  font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
`
export default Wrapper
