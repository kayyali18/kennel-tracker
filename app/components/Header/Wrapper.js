import styled from 'styled-components';

const Wrapper = styled.nav`
  display: flex;
  justify-content: space-around;
  flex-wrap: wrap;
  background-color: #fff;
  height: 100px;
  position: sticky;
  top: 0;
`;

export default Wrapper;
