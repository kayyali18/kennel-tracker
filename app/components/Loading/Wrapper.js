import styled from 'styled-components'

const DotWrapper = styled.div`
  display: flex;
  align-items: center;
  justify-content: center;
`

export default DotWrapper
