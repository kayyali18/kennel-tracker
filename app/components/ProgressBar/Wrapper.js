import styled from 'styled-components'

const Wrapper = styled.div`
  width: 50vw;
  margin: auto;
  height: 100px;
  background-color: green;
`

export default Wrapper
