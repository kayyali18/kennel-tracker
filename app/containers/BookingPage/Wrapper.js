import styled from 'styled-components'

const Wrapper = styled.main`
  min-height: 100vh;
  min-width: 100vw;

  display: grid;
`
export default Wrapper
