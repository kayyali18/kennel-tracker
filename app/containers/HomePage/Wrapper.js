import styled from 'styled-components'

const Wrapper = styled.section`
  display: flex;
  justify-content: center;
  align-items: center;
  height: 100vh;
  max-width: 100vw;
  background: linear-gradient(#a5a5a5, #f2f2f2);
`
export default Wrapper
