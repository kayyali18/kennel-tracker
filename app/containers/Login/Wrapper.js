import styled from 'styled-components'

const Wrapper = styled.section`
  margin: 220px auto;
  height: 200px;
  width: 500px;
  background: #fff;
  border: 1px solid grey;
`
export default Wrapper
