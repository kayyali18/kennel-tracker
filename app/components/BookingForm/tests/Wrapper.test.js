import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'

import Wrapper from '../Wrapper'

describe('<Wrapper />', () => {
  it('should match snapshot', () => {
    const renderedComponent = renderer.create(<Wrapper />).toJSON()
    expect(renderedComponent).toMatchSnapshot()
  })

  it('should render a article', () => {
    const renderedComponent = renderer.create(<Wrapper />).toJSON()
    expect(renderedComponent.type).toEqual('article')
  })
})
