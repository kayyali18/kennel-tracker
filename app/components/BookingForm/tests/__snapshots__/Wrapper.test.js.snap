// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Wrapper /> should match snapshot 1`] = `
.c0 {
  color: #fff;
  max-height: 70vh;
  overflow: auto;
}

.c0 input {
  border: 2px solid black;
}

.c0 .form-enter {
  margin-top: 30px;
  opacity: 0.01;
}

.c0 .form-enter-active {
  margin-top: 0px;
  opacity: 1;
  -webkit-transition: opacity 0.5s ease,margin 0.5s ease;
  transition: opacity 0.5s ease,margin 0.5s ease;
}

.c0 .form-leave {
  margin-top: 0px;
  opacity: 1;
}

.c0 .form-leave-active {
  margin-top: -30px;
  opacity: 0.01;
  -webkit-transition: opacity 0.3s ease,margin 0.5s ease;
  transition: opacity 0.3s ease,margin 0.5s ease;
}

<section
  className="c0"
/>
`;
